### PR TITLE
Tune up providers naming with what we have in DB

### PR DESCRIPTION
**Pull request by asmorodskyi**
* Related merge request: [gitlab.suse.de/qac/publiccloud-qa-suse-de!64](https://gitlab.suse.de/qac/publiccloud-qa-suse-de/-/merge_requests/64)

### CODE REVIEW

File: webui/settings.py [hunk 1]
```diff
@@ -137,33 +137,6 @@
 
 STATIC_URL = '/static/'
 
-logging.config.dictConfig({
```
Comment (pdostal) [inline webui/settings.py:140]: Was this just moved or also altered?

**Review by pdostal — APPROVED** [review]

**Review by ricardobranco777 — APPROVED** [review]
LGTM

File: webui/settings.py (hunk 1 above)
Comment (asmorodskyi) [reply webui/settings.py:140]: one line was altered 

```
-  'level': 'INFO',
+ 'level': loglevel,
```

Comment (asmorodskyi) [reply webui/settings.py:140]: from now on we will be able to tweak log level from pcw.ini w/o need to chage anything in PCW :eyeglasses:

**Pull request merged**